arrow-core/src/main/kotlin/arrow/core/extensions/eval/apply/EvalApply.kt

```diff
@@ -98,8 +93,8 @@ fun <A, B, Z> Kind<ForEval, A>.map2Eval(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "map(arg0, arg1, arg2)",
-  "arrow.core.Eval.map"
+    "Eval.mapN(arg0, arg1, arg2)",
+    "arrow.core.Eval"
```

Comment:
```suggestion
    "Eval.mapN(arg0, arg1) { a, b -> arg2(Tuple2(a, b)) }",
    "arrow.core.Eval", "arrow.core.Tuple2"
```

Idem for other `map` & `mapN` methods.